src/components/menu/js/menuServiceProvider.js

```diff
@@ -292,6 +292,10 @@ function MenuProvider($$interimElementProvider) {
               opts.mdMenuCtrl.close(false, { closeAll: true });
               handled = true;
               break;
+            case $mdConstant.KEY_CODE.TAB:
+              opts.mdMenuCtrl.close(false, { closeAll: true });
+              handled = false;
```

Comment:
Add comment that explains why `handled = false` here.

Reply:
Done.